src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const everyDayCost = 40;
+  const sevenDayCost = 50;
+  const alternativeCost = 20;
+
+  if (days < 3) {
+    return days * everyDayCost;
+  }
+
+  if (days >= 3 && days < 7) {
```

Comment:
To make this line smaller, you can go from largest to smallest 🙂
` if (days >= 7) {
return totalRent- sevenDayDiscount;
}

if (days >= 3) {
return totalRent - alternativeCost;
}`

Reply:
thank you for the comments, i will take them into account😊